nemo/collections/common/parts/__init__.py

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from nemo.collections.common.parts.data_utils import *
 from nemo.collections.common.parts.multi_layer_perceptron import MultiLayerPerceptron
 from nemo.collections.common.parts.transformer_utils import *
+from nemo.collections.common.parts.utils import *
```

Comment:
could you please remove *

Reply:
why? those are utils functions not classes and will grow with time, enumerating all is error prone

Reply:
I just noticed @blisc started pushing for this in asr or are we fine with * in __init__.py ? @okuchaiev ?

Reply:
let's merge now, since it's a blocker for other PR, and update later if needed